packages/raystack/calendar/calendar.tsx

```diff
@@ -1,21 +1,36 @@
-import { DayPicker, DayPickerProps, DropdownProps } from "react-day-picker";
-import { cva } from "class-variance-authority";
+import {
+  dateLib,
+  DayPicker,
+  DayPickerProps,
+  DropdownProps,
+} from 'react-day-picker';
+import { cva } from 'class-variance-authority';
 import {
   ChevronRightIcon,
   ChevronLeftIcon,
   ChevronUpIcon,
   ChevronDownIcon,
-} from "@radix-ui/react-icons";
-import styles from "./calendar.module.css";
-import { Select } from "~/select";
-import { ChangeEvent, useEffect, useState } from "react";
-import { Flex } from "~/flex/flex";
+} from '@radix-ui/react-icons';
+import styles from './calendar.module.css';
+import { Select } from '~/select';
+import { ChangeEvent, useEffect, useState } from 'react';
+import { Flex } from '~/flex/flex';
+import { Tooltip } from '~/tooltip';
+import Skeleton from 'react-loading-skeleton';
 
 interface OnDropdownOpen {
   onDropdownOpen?: VoidFunction;
 }
 
-export type CalendarProps = DayPickerProps & OnDropdownOpen;
+interface CalendarPropsExtended {
+  showTooltip?: boolean;
+  tooltipMessages?: { [key: string]: any };
```

Comment:
any?

Reply:
kept the type open so that we can pass any relevant value for the message